src/command.coffee

```diff
@@ -444,6 +444,9 @@ forkNode = ->
     cwd:        process.cwd()
     env:        process.env
     stdio:      [0, 1, 2]
+  ['SIGINT', 'SIGTERM'].forEach (signal) ->
```

Comment:
I thought so looking at the surrounding code but in this case the new scope is necessary for the code to work properly.

Anyway I retargeted against `2` and used `for...in` with `do` for the scope.